common/src/helpers/router.tsx

```diff
@@ -157,3 +157,41 @@ export const Link: React.FC<{ href: string; title: string }> = ({
     {children}
   </a>
 );
+
+interface ScrollToTopProps {
+  /** The top position of element relative to the viewport.  */
+  offsetTop?: number;
+  /** Set the scroll behavior to 'auto'. Default = 'smooth' */
+  scrollBehaviorAuto?: boolean;
+  children: React.ReactElement | null;
+}
+
+export const ScrollToTop: React.FC<ScrollToTopProps> = ({
+  offsetTop,
```

Comment:
minor tip! if you define default value for offsetTop here `offsetTop=0` , you could eliminate the `if else`  below

Reply:
> minor tip! if you define default value for offsetTop here offsetTop=0 , you could eliminate the if else below

Thanks @brindasasi! I was trying to figure out a way to simplify or get rid of the conditional statements  and this solution is succinct.